insta_reels_publishing_api_sample/index.js

```diff
@@ -63,7 +82,12 @@ app.get("/login", function (req, res) {
 // Callback route for handling FB OAuth user token And reroute to '/pages'
 app.get("/callback", async function (req, res) {
     const code = req.query.code;
-    const uri = `https://graph.facebook.com/oauth/access_token?client_id=${APP_ID}&redirect_uri=${REDIRECT_URI}&client_secret=${API_SECRET}&code=${code}`;
+    const uri = buildGraphAPIURL('oauth/access_token', {
+        client_id: APP_ID,
+        redirect_uri: REDIRECT_URI,
+        client_secret: API_SECRET,
+        code,
```

Comment:
Shouldn't this be `code: code`?

Reply:
They are equivalent, I am using the [shorthand notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions).